source/tests/consistent/common.py

```diff
@@ -365,7 +365,7 @@ def tearDown(self) -> None:
             clear_session()
 
 
-def parameterized(*attrs: tuple) -> Callable:
+def parameterized(*attrs: tuple, **kwargs: Dict[str, tuple]) -> Callable:
```

Comment:
If `**` is used, the type annotation should not be `dict`. See https://github.com/deepmodeling/deepmd-kit/pull/3792#issuecomment-2116830763